test/jdk/com/sun/jdi/EATests.java

```diff
@@ -1999,7 +1999,7 @@ public void warmupDone() {
         // Use new lock. lockInflatedByContention might have been inflated because of recursion.
         lockInflatedByContention = new XYVal(1, 1);
         // Start thread that tries to enter lockInflatedByContention while the main thread owns it -> inflation
-        DebuggeeWrapper.newThread(() -> {
+        TestScaffold.newThread(() -> {
```

Comment:
Is this neccessary?

Reply:
Yes, DebuggeeWrapper doesn't exist in 21. I believe it's a rename from 'TestScaffold'.

Reply:
Yes: https://github.com/openjdk/jdk/commit/ee9776fa23e2287b704d4f1a55179e83516d1e4c